lms/djangoapps/instructor/tests/test_legacy_raw_download_csv.py

```diff
@@ -67,14 +67,17 @@ def test_download_raw_grades_dump(self):
 '''
         self.assertEqual(body, expected_csv, msg)
 
-    def test_grade_summary_data(self):
+    def get_expected_data(
+        self, get_grades=True, get_raw_scores=False, use_offline=False,
+    ):
         """
-        Test grade summary data report generation
-        """
-        self.answer_question()
+        Return expected results from the get_student_grade_summary_data call
+        with any options selected.
 
-        request = DummyRequest()
-        data = get_student_grade_summary_data(request, self.course, get_raw_scores=False)
+        Note that the kwargs accepted by get_expected_data (and their
+        default values) must be identical to those in
+        get_student_grade_summary_data.
+        """
```

Comment:
I'm puzzled by this comment, though it might be simply a lack of understanding.  Why must the `kwargs` be the same?  Do we need a test for when they're not?